tests/e2e/utils/delete-auth-cookie.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Deletes authentication cookies to sign out the current user.
+ */
+export async function deleteAuthCookie() {
```

Comment:
Apologies if this wasn't clear, but we also want to update the `logoutUser` utility to use this as well rather than duplicate it.